geode-gfsh/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/GfshHostNameVerificationDistributedTest.java

```diff
@@ -60,14 +62,19 @@ public void setupCluster() throws Exception {
         .isCA()
         .generate();
 
-    CertificateMaterial locatorCertificate = new CertificateBuilder()
+    String hostname = LocalHostUtil.getCanonicalLocalHostName();
+    final CertificateBuilder builder = new CertificateBuilder()
         .commonName("locator")
         .issuedBy(ca)
         .sanDnsName(InetAddress.getLoopbackAddress().getHostName())
-        .sanDnsName(InetAddress.getLocalHost().getHostName())
-        .sanIpAddress(InetAddress.getLocalHost())
-        .sanIpAddress(InetAddress.getByName("0.0.0.0")) // to pass on windows
-        .generate();
+        .sanDnsName(hostname)
```

Comment:
these 2 lines should be put in "else" or some better place?

Reply:
loopback is supposed to be there.  The hostnamechecker in the jdk will ignore the sanDnsName(hostname) entry if it's an IP literal, so it doesn't hurt to have it there.